Source_Code/Common/Messages/StatusUpdates/PlayerHandUpdate.java

```diff
@@ -0,0 +1,24 @@
+package Common.Messages.StatusUpdates;
+
+import Common.*;
+
+public class PlayerHandUpdate extends StatusUpdate
+{
+   private String playerHand;

```

Comment:
Why not leave the hand as a hand? The client will need it in hand-form, so we may as well just send it as a hand already